store/BasalTestingSlice.js

```diff
@@ -0,0 +1,80 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+
+export const addBasalTest = createAsyncThunk(
+  'addTest',
+  async ({ numTest, glucose,date }) => {
+    try {
+      const jwt = await AsyncStorage.getItem('token');
+      const res = await axios.post(
+        'http://localhost:3000/api/auth/signin',
+        {
+          numTest: numTest,
+          glucose: glucose,
+          date: date
+        },
+        {
+          header: {Authorization: `Bearer ${jwt}`}
+        }
+      );
+      return res.data;
+    } catch (err) {
+      console.error(`Error!: ${err}`);
+    }
+  }
+);
+
+export const deleteBasalTest = createAsyncThunk(
+  'deleteTest',
+  async ({ id }) => {
+    try {
+      const res = await axios.post(
+        'http://localhost:3000/api/auth/signup',
+        {
+            headers: {Authorization: `Bearer ${jwt}`}
+        });
+      return res.data;
+    } catch (err) {
+      alert('Whoops! Something went wrong.');
+      console.error(`Error!: ${err}`);
```

Comment:
let me know if you are able to replace console.error with something else in the catch blocks, i was trying to remove console logs in my app